sage-ios/SAGE/SAGE/AnnouncementsDetailView.swift

```diff
@@ -0,0 +1,116 @@
+//
+//  AnnouncementsDetailView.swift
+//  SAGE
+//
+//  Created by Erica Yin on 11/7/15.
+//  Copyright © 2015 Cal Blueprint. All rights reserved.
+//
+
+import Foundation
+
+class AnnouncementsDetailView: UIView {
+    var tableView = UITableView()
```

Comment:
always bounce vertical lets you always scroll it up and down no matter how tall (if it's too short itll just do that rubber bandy thing that all tableviews always do)